web-services/query/src/main/java/datawave/webservice/query/configuration/GenericQueryConfiguration.java

```diff
@@ -167,6 +170,48 @@ public void setBypassAccumulo(boolean bypassAccumulo) {
         this.bypassAccumulo = bypassAccumulo;
     }
     
+    /**
+     * @return - the accumulo password
+     */
+    public String getAccumuloPassword() {
+        return this.accumuloPassword;
+    }
+    
+    /**
+     * Sets configured password for accumulo access
+     *
+     * @param password
+     *            the password used to connect to accumulo
+     */
+    public void setAccumuloPassword(String password) {
+        this.accumuloPassword = password;
+    }
+    
+    /**
+     *
+     * @return the accumulo password env target
+     */
+    public String getAccumuloPasswordEnv() {
+        return accumuloPasswordEnv;
+    }
+    
+    /**
+     * Sets the accumulo password from the provided env target. The value set in the environment will always overwrite any preconfigured value.
+     *
+     * @param accumuloPasswordEnv
+     *            the environment variable where the accumulo password is stored
+     */
+    public void setAccumuloPasswordEnv(String accumuloPasswordEnv) {
+        this.accumuloPasswordEnv = accumuloPasswordEnv;
+        if (StringUtils.isNotBlank(accumuloPasswordEnv)) {
+            String password = System.getenv(accumuloPasswordEnv);
+            if (StringUtils.isNotBlank(password)) {
+                this.accumuloPassword = password;
```

Comment:
splitting hairs, but would it make sense to use the setter for the accumulo password rather then directly modding it here?

Reply:
works for me. I'll add it in the next commit